config/www/cgi/script.py

```
@@ -0,0 +1,59 @@
+        button {{ padding: 10px 20px; font-size: 16px; }}
+        h1 {{ color: #333; }}
+    </style>
+</head>
+<body>
+    <h1>Button clicked {} times.</h1>
+    <form action="script.py" method="post">
+        <button type="submit">Click Me</button>
+    </form>
+</body>
+</html>
+""".format(count))
```

Ruff

Use f-string instead of `format` call

Convert to f-string

(UP032)